upgrade-tests/src/test/java/io/zeebe/test/UpgradeTest.java

```diff
@@ -152,6 +161,36 @@ private static BpmnModelInstance jobWorkflow() {
         state.client().newCompleteCommand(key).send().join();
   }
 
+  private static BpmnModelInstance msgWorkflow() {
```

Comment:
IMO: a full name is easier to read. So `messageWorkflow()` instead of `msgWorkflow()`. But it is more of a personal preference.